subcommands/events/create.go

```diff
@@ -0,0 +1,56 @@
+package events
+
+import (
+	"io/ioutil"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	cmd.AddCommand(&cobra.Command{
+		Use:   "mk-push <label> <url>",
+		Short: "Create an event queue that will ingest events at the URL",
+		Args:  cobra.ExactArgs(2),
+		Run:   doCreatePush,
+	})
+
+	cmd.AddCommand(&cobra.Command{
+		Use:   "mk-pull <label> <creds file>",
```

Comment:
Do we need to comment/explain to users what `<creds file>` mean?

Reply:
This can be confused with the TUF `<creds file>`; I'd be more verbose here e.g. `<oauth creds file to save to>` (or what is it?).

Reply:
commit 2a3de1e7598590f507f05c0d729f6d4f2df7826a